chat-widget/src/common/telemetry/TelemetryConstants.ts

```diff
@@ -109,6 +109,7 @@ export enum TelemetryEvent {
     OnNewMessageFailed = "OnNewMessageFailed",
     OnNewMessageAudioNotificationFailed = "OnNewMessageAudioNotificationFailed",
     DownloadTranscriptResponseNullOrUndefined = "DownloadTranscriptResponseNullOrUndefined",
+    ErrorUILoaded = "ErrorUILoaded",
```

Comment:
Can we change this to ErrorUIPaneLoaded? In case there will be addition "error UI"